App.tsx

```diff
@@ -1,40 +1,28 @@
-import React from 'react';
 import { I18nextProvider } from 'react-i18next';
-import { LogBox, AppState } from 'react-native';
+import { LogBox } from 'react-native';
 import { RootSiblingParent } from 'react-native-root-siblings';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
-import { QueryClientProvider, focusManager } from 'react-query';
 
 import i18nConfig from './src/config/i18n';
-import queryClient from './src/config/queryClient';
+import { AuthProvider } from './src/context/AuthContext';
+import { QueryClientProvider } from './src/context/QueryClientContext';
 import AppNavigator from './src/navigation/AppNavigator';
 
-focusManager.setEventListener(() => {
-  const handleAppStateChange = (appState: any) => {
-    console.log('AppState: ', appState);
-    focusManager.setFocused(appState === 'active');
-  };
-
-  const appState = AppState.addEventListener('change', handleAppStateChange);
-
-  return () => {
-    appState.remove();
-  };
-});
```

Comment:
I've moved it to the query client context, but it's not enabled yet. What does it do? @alvrba

Reply:
It only `consle.log` the app current state (active, idle...), so it is not necessary and has no real functionality